planners/src/solver.rs

```diff
@@ -0,0 +1,272 @@
+use crate::encode::{encode, populate_with_task_network, populate_with_template_instances};
+use crate::fmt::{format_hddl_plan, format_partial_plan, format_pddl_plan};
+use crate::forward_search::ForwardSearcher;
+use crate::solver::Strat::{Activity, Forward};
+use crate::Solver;
+
+use anyhow::{Context, Result};
+use aries_model::state::Domains;
```

Comment:
Possibility of missing `state.rs` in `master` branch.

https://github.com/plaans/aries/tree/master/model/src